File: py-codellama13B-AWQ-all/taskid-36_fizz_buzz-gen17.py
def fizz_buzz(n: int) -> int:
    """Return the number of times the digit 7 appears in integers less than n which are divisible by 11 or 13.
    >>> fizz_buzz(50)
    0
    >>> fizz_buzz(78)
    2
    >>> fizz_buzz(79)
    3
    """


    # Solution 1
    # count_7 = 0
    # for i in range(1, n):
    #     if i % 11 == 0 or i % 13 == 0:
    #         count_7 += str(i).count('7')
    # return count_7

    # Solution 2
    count_7 = 0
    for i in range(1, n):
        if i % 11 == 0 or i % 13 == 0:
            while '7' in str(i):
                count_7 += 1
                i //= 10
    return count_7

File: java-codellama7B-FP-all/taskid-36_fizz_buzz-gen17.java
import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    // Return the number of times the digit 7 appears in integers less than n which are divisible by 11 or 13.
    // >>> fizzBuzz((50l))
    // (0l)
    // >>> fizzBuzz((78l))
    // (2l)
    // >>> fizzBuzz((79l))
    // (3l)
    public static long fizzBuzz(long n) {

        long count = 0;
        for (int i = 2; i < n; i++) {
            if (((i % 11 == 0) || (i % 13 == 0))) {
                String s = String.valueOf(i);
                for (int j = 0; j < s.length(); j++) {
                    if (s.charAt(j) == '7') {
                        count++;
                    }
                }
            }
        }
        return count;
    }
    public static void main(String[] args) {
    assert(fizzBuzz((50l)) == (0l));
    assert(fizzBuzz((78l)) == (2l));
    assert(fizzBuzz((79l)) == (3l));
    assert(fizzBuzz((100l)) == (3l));
    assert(fizzBuzz((200l)) == (6l));
    assert(fizzBuzz((4000l)) == (192l));
    assert(fizzBuzz((10000l)) == (639l));
    assert(fizzBuzz((100000l)) == (8026l));
    }

}
